lang/ghc/Portfile

```diff
@@ -118,10 +118,25 @@ if { [variant_isset "prebuilt"] } {
                     port:texlive-fonts-recommended \
                     port:texlive-latex-extra
 
+    post-extract {
+        # test: move extracted testsuite directory to src
+        xinstall -m 0755 -d ${srcpath}/${distname}
+        move        ${worksrcpath}/testsuite ${srcpath}/${distname}/testsuite
+    }
+
     pre-configure {
         xinstall -m 0755 -d ${bootstrap_dir}
     }
 
+    # fix iconv linking issues if port libiconv is installed
+    # issue: file `[lindex [glob -directory ${worksrcpath}/libraries/base/dist-install/build -type f libHSbase-*-ghc${version}.dylib] 0]`
+    # links to /usr/lib/libiconv.2.dylib, but ${prefix}/lib/libiconv.2.dylib
+    # purposely mangles symbol names
+    compiler.library_path \
+                    /usr/lib:${prefix}/lib
+    configure.ldflags \
+                    "-L/usr/lib -liconv ${configure.ldflags}"
```

Comment:
Simpler to write `configure.ldflags-prepend -L/usr/lib -liconv`

Is adding `-liconv` really needed? I assume the build was adding it on its own before.

Reply:
I replaced both flags with an explicit `/usr/lib/libiconv.dylib`, which is all that's necessary here.